src/main/java/website/frontrow/Launcher.java

```diff
@@ -49,7 +49,7 @@ public static void main(String[] args)
         // Initialize the Logger Class, so that it can Log actions taken.
         new Log();
 
-        String[] levels = {"/level/1.txt", "/level/2.txt", "/level/3.txt"};
+        String[] levels = { "/level/3.txt"};
```

Comment:
I assume you only loaded level3 for testing, right?

Please change it back to be able to load all 3 levels.